### ISSUE TITLE
Bump up minimum yarn version

### ISSUE THREAD

**Issue by skellock** [infinitered/ignite#1052]
Spent an hour with someone on the community site debugging an issue where `ignite new Thing` would create the directory and not put anything in it.

Turns out with his setup `node@7.7.1` and `yarn@0.21.3`, both installed via `brew` (and no permissions issues; nor `react-native` installed globally) didn't work.

We tried `--npm` which didn't work.

Logging with `--debug` didn't show anything (which surprises me! -- possibly a bug? certainly not working as intended).

The solution to his problem was `brew update && brew upgrade yarn`.

Let's add another check for that combo:

```js
if (node === '7.7.1' && yarn === '0.21.3') {
  console.warn('Good news!  We are about to save you a lot of time!  You should upgrade yarn.')
}
```

**Comment by skellock** [comment]
Related https://github.com/yarnpkg/yarn/issues/2871

**Comment by ruddell** [comment]
I happen to use those exact versions and haven't had any issues.  I installed yarn with npm, node with nvm, and use Mac.  I can update yarn/node, it just seems like a weird check to me.

<details>
  <summary>My ignite doctor output (Click to expand)</summary>
```
System
  platform           darwin                                                                          
  arch               x64                                                                             
  cpu                8 cores                    Intel(R) Core(TM) i7-4850HQ CPU @ 2.30GHz  
  directory          /private/tmp/ignite/Test                                                        

JavaScript
  node               7.7.1        /Users/ruddell/.nvm/versions/node/v7.7.1/bin/node  
  npm                4.1.2        /Users/ruddell/.nvm/versions/node/v7.7.1/bin/npm   
  yarn               0.21.3       /Users/ruddell/.nvm/versions/node/v7.7.1/bin/yarn  

React Native
  react-native-cli   2.0.1       
  app rn version     0.42.0      

Ignite
  ignite             2.0.0-beta.9   /Users/ruddell/.nvm/versions/node/v7.7.1/bin/ignite  

Android
  java               1.8.0_60     /usr/bin/java                           
  android home       -            /usr/local/Cellar/android-sdk/24.4.1_1  

iOS
  xcode              8.3.2   
```
</details>

**Comment by skellock** [comment]
Cheers @ruddell .  I wonder if yarn-from-brew vs. yarn-from-npm makes a difference?

**Comment by ruddell** [comment]
Brew is the recommended way to install yarn on Mac, installing via npm is actually [warned against](https://yarnpkg.com/lang/en/docs/install/#alternatives-tab).  

I don't think it's that big of a factor because Brew used to install Yarn with `npm` lol (until a month ago) https://github.com/Homebrew/homebrew-core/pull/11881

**Comment by jamonholmgren** [comment]
@skellock I'm not sure if this needs to be fixed before 2.0 release. Happy to tackle it later.

**Comment by jamonholmgren** [comment]
I haven't seen issues with Yarn in a long time. Closing this.
